### ISSUE TITLE
Is there a CLI for segmenting a single image?

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ivadomed#1002]
I'm trying to test a model, and I just want to test it on a single image.

I see that we have the following API:

https://github.com/ivadomed/ivadomed/blob/ea01544efc411bda339137de4914b63bb96345b4/ivadomed/inference.py#L314

is this accessible via CLI?

**Comment by mariehbourget** [comment]
No, it is not accessible via CLI.

At the moment, the 2 options are:

- use the `--segment` command via CLI. However, it is not optimal for a single image as it requires the images to be in a bids-compliant dataset.
- use the API in a script. Example [here](https://github.com/neuropoly/axondeepseg/issues/523#issuecomment-887062124) that I used previously for microscopy.

**Comment by jcohenadad** [comment]
Thank you @mariehbourget. How about creating a new CLI script what has [this code](https://github.com/neuropoly/axondeepseg/issues/523#issuecomment-887062124) and that would be called something like `ivadomed_segment_image`

**Comment by mariehbourget** [comment]
Yes I agree, that is missing at the moment and would be quite useful when models are not yet integrated in SCT or ADS.

**Comment by jcohenadad** [comment]
Another incentive to having a separate segmentation CLI is that the config file might be cropping the input image, and we might want full image input during prediction. IIUC, this is not possible with the current training/testing workflow, which relies on the _same_ input config file.
